### PR TITLE
test for sort table

### PR DESCRIPTION
**Pull request by SetterVet**

### CODE REVIEW

File: spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
     context 'for login' do
       let(:user) { FactoryBot.create(:user) }
       it 'returns valid current_user and status 200' do
-        login_user(user)
+        login_user_by_devise(user)
```
Comment (VasylYaremkiv) [inline spec/controllers/users_controller_spec.rb]: rename back to login_user

File: spec/feature/select_project_spec.rb [hunk 2]
```diff
@@ -11,9 +11,7 @@
 
   before(:each) do
     visit new_user_session_path
-    fill_in 'Email', with: first_user.email
-    fill_in 'Password', with: first_user.password
-    click_button 'Log in'
+    login_user_by_factory(first_user)
```
Comment (VasylYaremkiv) [inline spec/feature/select_project_spec.rb]: rename to `sign_in_user`

File: spec/feature/sort_users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe 'Sort users', type: :feature do
+  let!(:first_user) { FactoryBot.create(:user, last_name: 'bbb', first_name: 'cc') }
+  let!(:second_user) { FactoryBot.create(:user, last_name: 'ccc', first_name: 'aa') }
+  let!(:third_user) { FactoryBot.create(:user, last_name: 'aaa', first_name: 'bb') }
+
+  before(:each) do
+    visit new_user_session_path
```
Comment (VasylYaremkiv) [inline spec/feature/sort_users_spec.rb]: move `visit new_user_session_path` to sign_in_user method

Comment (SetterVet) [reply spec/feature/sort_users_spec.rb]: renamed @VasylYaremkiv

**Pull request merged**